wallet/test/Test/Pos/Util/MnemonicSpec.hs

```diff
@@ -65,31 +66,38 @@ spec = do
         it "Can generate 128 bits entropy" $
             (length . entropyToByteString <$> genEntropy @128) `shouldReturn` 16
 
-        it "Mnemonic to JSON" $ forM_ testVectors $ \(bytes, _, mnemonic, _, _) ->
+        it "Mnemonic to JSON" $ forM_ testVectors $ \(bytes, _, _, mnemonic, _, _) ->
```

Comment:
@KtorZ what about record wildcards instead of the `TestVector` tuple?

Reply:
I'd prone lenses over wildcards if anything, but yeah. As always, it started as a tuple with 2 elements aaaaand... now we have this little monster.
Feel free to refactor that part; this will also re-trigger the failing AppVeyor CI :/